igvc_rviz_plugins/src/sensor_panel.cpp

```diff
@@ -0,0 +1,171 @@
+#include <igvc_rviz_plugins/sensor_panel.h>
+#include <pluginlib/class_list_macros.h>
+#include <QVBoxLayout>
+//#include <std>
+//#include "rjTimer.h"
+
+/*
+ * Just as in the header, everything needs to happen in the igvc_rviz_plugins namespace.
+ */
+namespace igvc_rviz_plugins
+{
+sensor_panel::sensor_panel(QWidget *parent) : rviz::Panel(parent)  // Base class constructor
+{
+  // Panels are allowed to interact with NodeHandles directly just like ROS nodes.
+  ros::NodeHandle nh;
+
+  // Initialize a label for displaying some data
+
+  labels[0] = new QLabel("IMU: Placehold\n");
```

Comment:
see previous comment about converting to map

Reply:
Done. Pushed.